docs/how-to/ide-setup.txt

```diff
@@ -0,0 +1,112 @@
+.. _ide-setup:
+
+#####################
+ IDE Setup (PyCharm)
+#####################
+
+Model developers may develop and debug models with their preferred IDE.
+In this guide, we show how to configure PyCharm to develop and debug
+against Determined's :ref:`Native API <tutorials_native-api>`.
```

Comment:
Can we add a bit of clarifying language about why the native API is different than the trial API in this respect / what you need to do (nothing?) if using the trial API.

Reply:
yes I took a pass at clarifying this